.github/workflows/format.yml

```
@@ -0,0 +1,35 @@
+name: Format
+
+on:
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  format:
+    if: |
+      github.event_name == 'pull_request_review' && 
+      github.event.review.state == 'commented' && 
+      startsWith(github.event.review.body, '/') &&
+      contains(fromJSON(vars.COMMANDS_TO_FORWARD), github.event.review.body)
```

This if only for format job, so can be simplified to 

```suggestion
      startsWith(github.event.review.body, '/prettify')
```